### DISCUSSION TITLE
Dynamic theme based on viewer's github theme?

### DISCUSSION THREAD

**Discussion by Mario-paul** [anuraghazra/github-readme-stats#1880] [Q&A] [answered]
How can you make it so someone viewing our readme page sees the cards themed based on their Github theme? For instance, someone viewing my user page with the default Github light theme would get Default (white) readme-stats theme cards, but someone with Dark or High Contrast Github theme would see darkcula or monokai or darkowl themed cards?

**Comment by rickstaa** [comment, marked as answer]
@Mario-paul Thanks for your question. This topic has been discussed on #1415. I use the solutions given there on my own [README](https://github.com/rickstaa). Feel free to fork it and check it out.

**Reply by Mario-paul** [reply]
@rickstaa thank you chief, that's exactly what I was looking for.

Question, in that thread you mentioned the new and official way to dynamically theme stuff, 

> As of today, this also is possible using the HTML \<picture> element in combination with the prefers-color-scheme media feature. It is also (for more information, see https://github.blog/changelog/2022-05-19-specify-theme-context-for-images-in-markdown-beta/). Please, however, be aware that in contrast to the method above, this method does not work when people switch the theme without reloading the page.

given the downside of it, is there any reason to use the new way instead of the original method?

**Reply by rickstaa** [reply]
@Mario-paul No, it doesn't matter. I chose the old method in my readme since I didn't yet have time to change it. Can you please mark my answer if it solved your question? 🙂

**Comment by rickstaa** [comment]
All the currently available ways for creating a dynamic theme are now documented in the [README.md](https://github.com/anuraghazra/github-readme-stats#responsive-card-theme) (See https://github.com/anuraghazra/github-readme-stats#responsive-card-theme).
